lib/kafo/configuration.rb

```diff
@@ -88,6 +88,10 @@ def modules
       @modules ||= @data.keys.map { |mod| PuppetModule.new(mod, PuppetModule.find_parser, self).parse }.sort
     end
 
+    def module(name)
+      modules.find { |m| m.name == name }
```

Comment:
could you please use the same method at https://github.com/domcleal/kafo/blob/e572fe236f19316f7ab97b12ac65d39831e3743d/lib/kafo/configuration.rb#L192